web/src/components/data-fields/ScrollableFormFieldWrapper.tsx

```diff
@@ -4,7 +4,7 @@ interface Props {
   fieldName: string;
 }
 
-export const FormationField = (props: React.PropsWithChildren<Props>): ReactElement => {
+export const ScrollableFormFieldWrapper = (props: React.PropsWithChildren<Props>): ReactElement => {
```

Comment:
I refactored this component because the scrollable functionality is used in different parts of the application. And this is part of a separate commit.